tests/drivers/build_all/dts_fixup.h

```diff
@@ -337,4 +337,11 @@
 #define DT_INST_0_MICROCHIP_ENC28J60_SPI_MAX_FREQUENCY	0
 #endif
 
+#ifndef DT_INST_0_TI_TMP116_LABEL
+#define DT_INST_0_TI_TMP116_LABEL ""
+#define DT_INST_0_TI_TMP116_BASE_ADDRESS 0
+#define DT_INST_0_TI_TMP116_BUS_NAME ""
+#define DT_TI_TMP116_BUS_I2C 1
+#endif
```

Comment:
Move these defines to the `#if defined(CONFIG_HAS_DTS_I2C)` section. They're currently in the SPI section

Reply:
Done.